lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -486,6 +486,22 @@ class Simulator {
     };
   }
 
+  /**
+   * @param {number} wastedBytes
+   */
+  simulateTimespan(wastedBytes) {
```

Comment:
I wanted to move the logic here for two reasons:
- Eventually we may want a true timespan simulator to calculate more detailed savings.
- It was easier to pipe the `observedThroughput` here.

I won't die on this hill though, we can also consider @brendankenny's design: https://github.com/GoogleChrome/lighthouse/pull/13323#issuecomment-961531948

Reply:
don't love this name since 'timespan' is generally something else in a lighthouse context.

i still like the `computeWastedMsWithWastedBytes` name tbh. how bout dat

Reply:
Ya SGTM

Reply:
> I wanted to move the logic here for two reasons:
> - Eventually we may want a true timespan simulator to calculate more detailed savings.
> - It was easier to pipe the `observedThroughput` here.
>
> we can also consider @brendankenny's design: https://github.com/GoogleChrome/lighthouse/pull/13323#issuecomment-961531948

Yeah the difference is mostly just where it's defined. There's always the danger of YAGNI, but in addition to it being a simple solution:
- `Simulation.Options` is where the other network analysis values are already passed in
- we only have one place where we construct a simulator and it's been like that for a long time and no plans to change it

so it works for me.

If we switch `devtools` throttling to use the observed throughput (as discussed in #13323) it may make more sense to do this overriding of throughput in `load-simulator` and only have the single `throughput` option, but we can think about that when/if that change comes up.